File: solita-dev-academy-2023-server/Models/Station.cs
namespace solita_dev_academy_2023_server.Models
{
    public class Station
    {
        public string Id { get; set; }
        public string Name_fi { get; set; }
        public string Name_se { get; set; }
        public string Name_en { get; set; }
        public string Address_fi { get; set; }
        public string Address_se { get; set; }
        public string Operator { get; set; }
        public int? Capacity { get; set; }
        public decimal? X { get; set; }
        public decimal? Y { get; set; }
    }
}

File: solita-dev-academy-2023-client/src/Models/Station.ts
export default interface Station {
    Id: string;
    Name_fi: string;
    Name_se: string;
    Name_en: string;
    Address_fi: string;
    Address_se: string;
    Operator: string;
    Capacity: number | null;
    X?: number;
    Y?: number;
}